bug-1292234-test-we/web-ext-config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  run: {
+    // Points to my local artifact development build of Firefox
+    firefox: '/Users/bdanforth/src/mozilla-unified/objdir-frontend-debug-artifact/dist/Nightly.app/Contents/MacOS/firefox',
```

Comment:
This seem pretty specific for your setup :)

Reply:
Is that bad? How might I abstract that away? One way or the other, I need to point to my local build.